providers/cnr/CNRProvider.go

```diff
@@ -0,0 +1,81 @@
+// Package CNR implements a registrar that uses the CNR api to set name servers. It will self register it's providers when imported.
```

Comment:
Please rename this file to be `cnrProvider.go` to be consistent with the others.